### PR TITLE
RCIAM SimpleSamlPHP v1.18

### PR DESCRIPTION
**Pull request by Tas-sos**
First approach for adding version `1.18`.

With variables :
* `ssp_version: "1.18"`
* `ssp_repo_version: "rciam-1.18"`
* `ssp_version_suffix: "rciam-1.18"`

New template files ( just copied from the old `1.17` version )
* [ ] `roles/ssp/templates/config/authsources-1.18.php.j2`
* [X] `roles/ssp/templates/config/config-metarefresh-1.18.php.j2`
* [X] `roles/ssp/templates/metadata/saml20-idp-hosted-1.18.php.j2`

### CODE REVIEW

File: roles/ssp/templates/metadata/saml20-idp-hosted-1.18.php.j2 [hunk 1]
```diff
@@ -0,0 +1,262 @@
+<?php
```
Comment (nikosev) [inline roles/ssp/templates/metadata/saml20-idp-hosted-1.18.php.j2:1]: Add an empty line after `<?php`

**Review by nikosev — COMMENTED** [review]
- There are not new changes for config-metarefresh-1.18.php.j2.
- For authsources-1.18.php.j2 there have been added new examples (that are commented), so no new changes are required for this template
- For saml20-idp-hosted-1.18.php.j2 only the empty line is added in SSP v1.18

File: roles/ssp/templates/metadata/saml20-idp-hosted-1.18.php.j2 (hunk 1 above)
Comment (Tas-sos) [reply roles/ssp/templates/metadata/saml20-idp-hosted-1.18.php.j2:1]: OK! Just an empty line added! Thanks!

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**